packages/commonwealth/client/scripts/views/components/LoadingIndicator/LoadingIndicator.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { CWText } from '../component_kit/cw_text';
+import CWCircleMultiplySpinner from '../component_kit/new_designs/CWCircleMultiplySpinner';
+import './LoadingIndicator.scss';
+
+type LoadingIndicatorProps = {
+  message?: string;
+};
+
+export const LoadingIndicator = (props: LoadingIndicatorProps) => {
```

Comment:
we have two generic components for loading 
- CWCircleMultiplySpinner
- CWCircleRingSpinner

We shouldn't create another component, especially that it just reuses generic loader. If you need text, just edit existing component so it accepts optional text

Reply:
This was a pre-existing component.  

It was used in like 5-10 places.

It uses CWCircleMultiplySpinner internally.

Generally I agree with you to keep things simple but I'd have to rewrite these 5-10 other uses.

I disagree though that we should change CWCircleMultiplySpinner or CWCircleRingSpinner.  Those should just be isolated/simple spinners.

The only other thing I did was wrap it in a 'screen' so that while a page is loading, we can just have a simple loading indicator in the center of the screen, with optional text.  This was also already present.

If we want to change it, we should do it in another PR since this was pre-existing.

Reply:
I don't insist to do it in this PR